_posts/2020-07-25-post-competition.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: news
+title: JAMDynamics steal victory in Student Robotics 2020
+---
+
+Congratulations to JAMDynamics who take victory in a tie-breaker match after a
+strong performance in the league and knockout stages.
+
+The Challenge: *Two Colours*
+----------------------------
+
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2020/arena.png"
+           caption="The virtual arena with its central raised platform" %}
+
+Our game this year, _Two Colours_, challenged teams retrieve gold or silver
+coloured tokens from on and around a raised platform in the centre of the arena.
+Having these tokens within their scoring zone (i.e. their corner of the arena)
+at the end of the match would earn them 3 points apiece. However, if a robot
+gathered more than one token colour within their scoring zone, each token's
+value drops from 3 points to 1 point. Full details of the game and the awards
+available this year are available in the [rulebook][rulebook].
+
+Our competition event this year was rather different to normal as COVID-19
+prevented our usual weekend event. Instead the competition took place over
+three weekends, with matches being live-streamed from within a simulator.
+While this changed many aspects of the competition the core challenge to create
+an autonomous robot remained the same.
+
+The Final
+---------
+
+This year's final featured four teams who had all proven their capabilities
+throughout the competition. All robots got off to a strong start, though
+JAMDynamics and Lawrence Sheriff School soon took a lead. However later in the
+match, the "poisoning" effect of having mixed tokens in a scoring zone changed
+everything leaving all four teams with the same number of points.
+
+<!-- Video of Final -->
+<iframe
+  class="center video"
+  src="https://www.youtube-nocookie.com/embed/XXX-TODO"
```

Comment:
May need to wait up to a day before the stream finishes processing and can be downloaded for cutting